src/main/java/securbank/models/Account.java

```diff
@@ -0,0 +1,184 @@
+package securbank.models;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import org.joda.time.LocalDateTime;
+
+/**
+ * 
+ * @author Madhu Illuri
+ */
+
+@Entity 
+@Table(name = "Account")
+public class Account {
+	
+	/** Account number is unique. */
+	@Id
+	@NotNull
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "accountNumber", nullable = false, unique = true, columnDefinition = "BINARY(16)")
+	private Long accountNumber;
+	
+	/** multiple account can be associated with an user	 */
+	@ManyToOne(cascade = CascadeType.ALL)
+	@NotNull
+	@JoinColumn(name = "user", unique = true, nullable = false)
+	private User user;
+	
+	/** balance on the account */
+	@NotNull
+	@Column(name = "balance", unique = false, nullable = false)
+	private Double balance;
+	
+	/** account type. Default an account will be checking(0), (1) for savings account	*/
+	@NotNull
+	@Column(name = "accountType", unique = false, nullable = false, columnDefinition = "int(1) DEFAULT '0' ")
+	private Integer accountType;
```

Comment:
@madhuilluri Lets keep it string instead of integer. Would be easier to interpret in DB as well as in service layer.